sysboundary/ionosphere.cpp

```diff
@@ -33,6 +33,7 @@
 #include "../vlasovmover.h"
 #include "../fieldsolver/fs_common.h"
 #include "../fieldsolver/fs_limiters.h"
+#include "../fieldsolver/ldz_magnetic_field.hpp"
```

Comment:
Why did this change?

Reply:
I't needs this to find the definition of propagateMagneticField.